core/templates/dev/head/domain/learner_dashboard/learner_dashboard_icons_directive.html

```diff
@@ -1,5 +1,5 @@
 <i class="oppia-learner-dashboard-icon fa fa-clock-o"
-   ng-show="canActivityBeAddedToLearnerPlaylist(getActivityId())"
+   ng-show="false"
```

Comment:
Why have you set it to false (always)?. Clicking on the clock symbol adds this activity to playlist. Setting this to false permanently doesn't allow allow the user to add the activity to playlist.

Reply:
@ishucr7 Thanks.I have added them for debugging,So that the Incomplete activity icon is always visible.I will remove them and submit them and submit the changes.